instrumentation/servlet/build.gradle.kts

```diff
@@ -1,9 +1,19 @@
 plugins {
   id("splunk.instrumentation-conventions")
+  id("splunk.muzzle-conventions")
+}
+
+muzzle {
+  pass {
+    group.set("javax.servlet")
+    module.set("servlet-api")
+    versions.set("[2.3,)")
+    assertInverse.set(true)
+  }
 }
 
 dependencies {
-  compileOnly("javax.servlet:servlet-api:2.2")
+  compileOnly("javax.servlet:servlet-api:2.3")
```

Comment:
Why this?

Reply:
We check for `javax.servlet.Filter` class in the `classLoaderMatcher()`, that class was added in 2.3.